plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  LinkButton,
+  EmptyState,
+  Link,
+  InfoCard,
+  Progress,
+  ErrorPanel,
+} from '@backstage/core-components';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+/**
+ * Props customizing the <FeaturedDocs/> component.
+ *
+ * @public
+ */
+export type FeaturedDocsProps = {
+  /** The entity filter used to display only the intended item/s */
+  filter: EntityFilterQuery;
+  /** An optional color which can be customized through themes */
+  color?: 'inherit' | 'primary' | 'secondary' | undefined;
```

Comment:
Is there a type defined by Material UI you could use here?

Reply:
I attempted to use a MUI type here, but since I am using other backstage core-components, there is a type error when using another type.